Loops_Lab.js

```diff
@@ -0,0 +1,210 @@
+// 1. Write a while loop and a for loop that takes the variable "num" and logs
+// all the numbers, descending, between "num" and 1.
+//
+let num = 20; // will display 19 to 2
+
+// started with the for loop first to don't have to reaffect num again
+for (let i = num - 1; i > 1; i--){
+  console.log(i);
+}
+console.log("----------------------------------------- \n");
+
+while (num > 2){
+  num --;
+  console.log(num);
+}
+console.log("----------------------------------------- \n");
+
+
+// 2. Write a while loop and a for loop that takes the variable "num", and
+// iterates over all numbers from 0 to "num". For each iteration, it will check
+// if the current number is even or odd, and log that to the screen
+// (e.g. "2 is even")
+//
+num = 20;
+
+for (i = 0; i <= num; i ++){
+  if (i % 2 === 0){
+    console.log(i, " is even");
+  } else console.log(i, " is odd");
```

Comment:
Even though this works for a single execution of code (in this case running only console.log), it is recommended practice to close the else statement with curly brackets.

Reply:
I got the same feedback from Alejo.
I knew about the curly brackets, just didn't know it's a good practice to always have them. I will make sure to remember that